### DISCUSSION TITLE
Keep citations "alive" when exporting to Word etc.

### DISCUSSION THREAD

**Discussion by mariashinoto** [quarto-dev/quarto-cli#5555] [Feature Requests]
I write all my academic work in RMarkdown or, recently, Quarto. Still, when collaborating with others, exporting to Word is usually necessary. When doing so, all citations get hard-coded; they are not any more "alive" as citations that can be modified. 

Having all citation units from Quarto transferred to typical Word-Citations, such that they can be modified in Word, would be a significant enhancement; the final polish can be done as if the whole document was created in Word. 

(I am working with Zotero and do not know whether Word treats different citation managers differently. Agreeing on Biblatex as a common denominator would be OK).

**Comment by cderv** [comment]
I think this would require support from upstream Pandoc as we leverage the Pandoc's citation processing for this. It seems there is not yet support for native Docx citation. See 
- https://github.com/jgm/pandoc/issues/7889

**Reply by mariashinoto** [reply]
Thanks for looking into it; I gave it a thumbs up at the link you provided.

**Comment by jeffkimbrel** [comment]
Commenting here because when searching on Google for similar (my search was "Quarto render keeping citations active for word") gives the simplistic suggestion that adding `citeproc: true` to the `quarto.yaml` file will fix this... but it doesn't appear to be true. While `citeproc` does appear to be valid for pandoc when working in quarto (https://quarto.org/docs/authoring/citations.html), it doesn't seem related to the larger issue.
